jovian/utils/commit.py

```diff
@@ -129,10 +129,18 @@ def commit(message=None,
         if project is None:
             log("Please provide the project argument e.g. jovian.commit(project='my-project')", error=True)
             return
+        if environment == "conda":
+            log("Colab uses a pip environment, cannot capture conda environment.", error=True)
+            return
```

Comment:
we can log the message here and change env to pip.

Reply:
Sure.

Reply:
![image](https://user-images.githubusercontent.com/24864161/95289292-ffca7e80-0887-11eb-84eb-bf1e93d4bf4a.png)